views/update/update-check-your-answers.html

```diff
@@ -20,8 +20,11 @@ <h1 class="govuk-heading-xl">{{ title }}</h1>
       {% include "includes/check-your-answers/verification-checks.html" %}
       <br>
       {% include "includes/entity.html" %}
-      <br>
-      {% include "includes/check-your-answers/beneficial-owner-statements.html" %}
+
+      {% if OE_CONFIGS.FEATURE_FLAG_ENABLE_UPDATE_STATEMENT_VALIDATION === "false" %}
```

Comment:
I wouldn't check for `=== 'false'` because the absence of the flag would mean this doesn't show.

At the top of the view I'd do

```
{% set updateTrustsEnabled = OE_CONFIGS.FEATURE_FLAG_ENABLE_UPDATE_STATEMENT_VALIDATION === "true" %}
```

Then here use

```
 {% if not updateTrustsEnabled %}
```

and below

```
 {% if updateTrustsEnabled %}
```

Reply:
Fixed